src/auth/routes.js

```diff
@@ -27,6 +28,7 @@ router.get(
   SpotifyConfig.authenticate('spotify', {
     failureRedirect: '/auth/spotify',
     successRedirect: '/graphql',
+    // successRedirect: '/home-page',
```

Comment:
sweet!  thanks for commenting here.  It helps me follow your code especially when you're more experienced in GraphQL than I am